tests/Woketo/Rfc6455/MessageHandler/PingFrameHandlerTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Test\Woketo\Rfc6455\MessageHandler;
+
+/**
+ * This file is a part of Woketo package.
+ *
+ * (c) Ci-tron <dev@ci-tron.org>
+ *
+ * For the full license, take a look to the LICENSE file
+ * on the root directory of this project
+ */
+
+use Nekland\Woketo\Rfc6455\Frame;
+use Nekland\Woketo\Rfc6455\FrameFactory;
+use Nekland\Woketo\Rfc6455\Message;
+use Nekland\Woketo\Rfc6455\MessageHandler\PingFrameHandler;
+use Nekland\Woketo\Rfc6455\MessageProcessor;
+use Nekland\Woketo\Utils\BitManipulation;
+use Prophecy\Argument;
+use React\Socket\ConnectionInterface;
+
+class PingFrameHandlerTest extends \PHPUnit_Framework_TestCase
+{
+    private $pingMessage;
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        // Normal ping frame without payload
+        $this->pingMessage = new Message();
+        $this->pingMessage->addFrame(new Frame(BitManipulation::hexArrayToString(['89', '00'])));
```

Comment:
The ping message is related to a specific case (the full message actually). So you should not set it in the setup part !